public/keymaps/c/clawsome_bookerboard_default copy.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Just realized this file doesn't have the correct filename.

```shell
git mv -- "public/keymaps/c/clawsome_bookerboard_default copy.json" "public/keymaps/c/clawsome_bookerboard_default.json"
git commit -m "renamed 'clawsome_bookerboard_default copy.json' to 'clawsome_bookerboard_default.json'"
```

Reply:
Are those commands I need to do, or did you just do them?

Reply:
You need to do these in Git through your command line interface.